src/components/NavigationAccount.vue

```diff
@@ -78,6 +78,10 @@ export default {
 			type: Object,
 			required: true,
 		},
+		inbox: {
```

Comment:
let's use `firstFolder` to avoid confusion later :)

Reply:
I have renamed it to firstFolder (and firstFolderRoute